tests/kani/Array/repeat_const_generic.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Ensure Kani can evaluate `[e; N]` when `N` is a const generic
+
+struct Foo<const N: usize> {
+    field: [u64; N],
+}
+
+impl<const N: usize> Foo<N> {
+    fn new() -> Self {
+        Self { field: [0; N] }
+    }
+}
+
+#[cfg(kani)]
+mod proofs {
+    use super::*;
+
+    #[kani::proof]
+    fn kani_crash() {
```

Comment:
Someone reading `kani_crash` may think the test is expect to crash Kani. Please rename the proof to avoid this.